services/movie.py

```diff
@@ -6,9 +7,14 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+        return queryset
```

Comment:
The early return statement here will prevent further filtering by genres or actors if a title is provided. Consider removing the return statement to allow combined filtering by title, genres, and actors.